corehq/ex-submodules/pillowtop/migrations/0008_sync_es_with_couch_webusers.py

```diff
@@ -0,0 +1,17 @@
+from django.db import migrations
+from django.core.management import call_command
+
+
```

Comment:
Do you think it would be fine to skip it on fresh installs?

Reply:
Right, will update!